xcode/Subconscious/SubconsciousTests/Tests_CombineUtilities.swift

```diff
@@ -20,6 +20,8 @@ final class Tests_CombineUtilities: XCTestCase {
         cancellables = Set()
     }
     
+    private static let TIMEOUT = 1.0
```

Comment:
These tests were timing out when run in parallel with the full test suite, seems like the timeout starts but the thread pool is busy. Anything below `0.5` fails on my machine with the full suite, running _just this file_ you only need `0.1`.

Reply:
Updated to camelCase